### ISSUE TITLE
[datepicker] active state set during calendar navigation

### ISSUE THREAD

**Issue by rsangiovanni** [angular-ui/bootstrap#5618]
### Bug description:
datepicker pop-up calendar set active a button different from current date:
* navigate days: first day the shown month is selected
* navigate months: current month is selected
* navigate years: 
  * for forward navigation: current year + 10xN (where N is number of the page forward) 
  * for backward navigation: current year - 10xN (where N is number of the page backward) 

### Link to minimally-working plunker that reproduces the issue:
UIBS demo page:  http://angular-ui.github.io/bootstrap/#/datepicker

### Version of Angular, UIBS, and Bootstrap
I found on my used version, but it still valid on latest version used on UIBS page

##### Angular:
* mine: 1.4.7
* UIBS page: 1.5.0

##### UIBS:
* mine: 0.43.0
* UIBS page: 1.2.4

##### Bootstrap:
* mine: 3.3.4
* UIBS page: 3.3.6

**Comment by wesleycho** [comment]
Can you phrase your bug description better? The text is hard to parse as to what you mean.

**Comment by wesleycho** [comment]
Closing due to confusion - feel free to open a new issue with clearer wording if this is an issue.
